envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadata.kt

```diff
@@ -221,11 +222,15 @@ private fun Value?.toSettings(defaultSettings: DependencySettings): DependencySe
             defaultSettings.retryPolicy
         )
     }
+    val routingPolicy = this?.field("routingPolicy")?.let { policy ->
+        mapToRoutingPolicy(value = policy, defaultRoutingPolicy = defaultSettings.routingPolicy)
+    }
 
     val shouldAllBeDefault = handleInternalRedirect == null &&
         rewriteHostHeader == null &&
         timeoutPolicy == null &&
-        retryPolicy == null
+        retryPolicy == null &&
+        routingPolicy == null
```

Comment:
We have to fix this. I'll create separate PR with sample solution

Reply:
What do you mean? Maybe this comment was intended to be on a line with a TODO comment?

Reply:
First problem: Statement is too large, 3 would be okey in one line but 5 or 6  
Second problem: We can't tell exactly which one is failing. Good luck debugging it.

Reply:
I agree, but let's do it in another PR like you said